spyder/plugins/variableexplorer/widgets/objectexplorer/objectexplorer.py

```diff
@@ -420,20 +420,28 @@ def _update_details_for_item(self, tree_item):
             data = attr_details.data_fn(tree_item)
             self.editor.setPlainText(data)
             self.editor.setWordWrapMode(attr_details.line_wrap)
+            # FIXME: The variable explorer is not tied to the code editor
+            # so these calls here probably should not be used, since it
+            # assumes there is an editor, which might not be the case at
+            # all. We can probably make a decission for the user and
+            # that should be good. Having all code editors in spyder in
+            # sync does not seem very useful in this case.
             show_blanks = CONF.get('editor', 'blank_spaces')
             update_scrollbar = CONF.get('editor', 'scroll_past_end')
```

Comment:
Let's make these options False in this PR.

Reply:
👍

Reply:
Fixed.